namechooser_3000.py

```diff
@@ -120,7 +129,7 @@ def get_random_number():
         keep_spinning = False
         
         print("\r                                    \n")
-        subprocess.call(['figlet', '-c', names[rnm%len(names)].decode('utf-8').encode('latin-1')])
+        subprocess.call(['figlet', '-c', names[rnm].decode('utf-8').encode('latin-1')])
```

Comment:
The invariant that rnm is within the range [0,len(names)-1] does not hold for all use cases. If we get into the exception case then we will call randint(1,len(names)) which will generate a number in the range [1,len(names)]. So we should fix that one as well to generate an expected number range.

Reply:
Right, missed that one.